package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start",
+    "start": "eslint --fix && next build && next start",
```

Comment:
Preciso que você volte com o comando original e deixe esse novo comando com outro alias. Pretendo utilizar o start para iniciar o servidor em produção no GCP e isso pode gerar conflitos e builds desnecessários

Reply:
vou deixar em outra script então, toda vez que mudo algo to precisando realizar o build